docs/settings.rst

```diff
@@ -599,6 +599,8 @@ Setting name (followed by default value, if any)    What does it do?
                                                     pagination.
 ``PAGINATION_PATTERNS``                             A set of patterns that are used to determine advanced
                                                     pagination output.
+``PAGINATION_FIXED = False``                        Set to true to have pages with fixed content, 
```

Comment:
In my opinion, `FIXED_PAGINATION` reads better than `PAGINATION_FIXED`. That being said, maybe a better way to describe this setting is `REVERSED_PAGINATION`? I'm not sure though.